### PR TITLE
Cover some mobile request commands

### PR DESCRIPTION
**Pull request by LevchenkoS**
Been done:
- Covered next mobile request commands:
  - `AlertRequest`
  - `AlertManeuverRequest`;
  - `ListFilesRequest`;
  - `ReadDidRequest`;
  - `SubscribeButtonRequest`;
- Improved `CommandsTest` and `CommandRequestTest` classes;
- Added missing methods to `MockMessageHelper`.

Related to: [APPLINK-24798](https://adc.luxoft.com/jira/browse/APPLINK-24798)

### CODE REVIEW

**Comment by LevchenkoS** [comment]
@AByzhynar, @okozlovlux, @Kozoriz, @LuxoftAKutsan, @VVeremjova, @vlantonov, @wolfylambova22, please review.

**Comment by vlantonov** [comment]
@LevchenkoS Reviewed.

File: src/components/application_manager/test/commands/mobile/alert_maneuver_request_test.cc [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+}
+
+TEST_F(AlertManeuverRequestTest, Run_IsWhiteSpaceExist_UNSUCCESS) {
+  MessageSharedPtr msg = CreateMessage(smart_objects::SmartType_Map);
+  (*msg)[am::strings::msg_params][am::strings::soft_buttons] = 0;
+  (*msg)[am::strings::msg_params][am::strings::tts_chunks] =
+      SmartObject(smart_objects::SmartType_Array);
+
+  SmartObject tts_chunk(smart_objects::SmartType_Map);
+  tts_chunk[am::strings::text] = "wrong chank syntax\t\n";
```
Comment (AByzhynar) [inline src/components/application_manager/test/commands/mobile/alert_maneuver_request_test.cc]: @LevchenkoS `chank->chunk`

File: src/components/application_manager/test/commands/mobile/alert_request_test.cc [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+
+TEST_F(AlertRequestTest, Run_AlertFrequencyIsTooHigh_UNSUCCESS) {
+  CommandPtr command(CreateCommand<AlertRequest>());
+
+  MockAppPtr app(CreateMockApp());
+  ON_CALL(app_mngr_, application(_)).WillByDefault(Return(app));
+
+  ON_CALL(*app, hmi_level())
+      .WillByDefault(Return(mobile_apis::HMILevel::HMI_BACKGROUND));
+  ON_CALL(*app, IsCommandLimitsExceeded(_, _)).WillByDefault(Return(true));
```
Comment (AByzhynar) [inline src/components/application_manager/test/commands/mobile/alert_request_test.cc]: @LevchenkoS `IsCommandLimitsExceeded->AreCommandLimitExceeded`

File: src/components/application_manager/test/commands/mobile/list_files_request_test.cc [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+TEST_F(ListFilesRequestTest, Run_TooManyHmiNone_UNSUCCESS) {
+  MockAppPtr app(CreateMockApp());
+  SharedPtr<ListFilesRequest> command(CreateCommand<ListFilesRequest>());
+
+  ON_CALL(app_mngr_, application(_)).WillByDefault(Return(app));
+  ON_CALL(*app, hmi_level())
+      .WillByDefault(Return(mobile_apis::HMILevel::HMI_NONE));
+
+  const uint32_t kListFilesInNoneAlowed = 1u;
```
Comment (AByzhynar) [inline src/components/application_manager/test/commands/mobile/list_files_request_test.cc]: @LevchenkoS `Alowed->Allowed`

**Comment by LuxoftAKutsan** [comment]
Reviewed

**Comment by AByzhynar** [comment]
@LevchenkoS Commit message edit: `MockMessageHalper -> MockMessageHelper`

File: src/components/application_manager/test/commands/mobile/list_files_request_test.cc [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+  ON_CALL(app_mngr_, application(_)).WillByDefault(Return(app));
+  ON_CALL(*app, hmi_level())
+      .WillByDefault(Return(mobile_apis::HMILevel::HMI_FULL));
+
+  ON_CALL(*app, increment_list_files_in_none_count()).WillByDefault(Return());
+
+  ON_CALL(*app, GetAvailableDiskSpace()).WillByDefault(Return(0));
+
+  am::AppFilesMap filesMap;
```
Comment (okozlovlux) [inline src/components/application_manager/test/commands/mobile/list_files_request_test.cc]: @LevchenkoS should it been named **files_map** ?

**Comment by okozlovlux** [comment]
@LevchenkoS  reviewed, but check variables names according to coding style

File: src/components/application_manager/test/commands/mobile/list_files_request_test.cc (hunk 4 above)
Comment (LevchenkoS) [inline src/components/application_manager/test/commands/mobile/list_files_request_test.cc]: @okozlovlux, fixed in 3190f78.

File: src/components/application_manager/test/commands/mobile/list_files_request_test.cc (hunk 3 above)
Comment (LevchenkoS) [inline src/components/application_manager/test/commands/mobile/list_files_request_test.cc]: @AByzhynar , fixed in 3190f78.

File: src/components/application_manager/test/commands/mobile/alert_request_test.cc (hunk 2 above)
Comment (LevchenkoS) [inline src/components/application_manager/test/commands/mobile/alert_request_test.cc]: @AByzhynar , fixed in 3190f78.

File: src/components/application_manager/test/commands/mobile/alert_maneuver_request_test.cc (hunk 1 above)
Comment (LevchenkoS) [inline src/components/application_manager/test/commands/mobile/alert_maneuver_request_test.cc]: @AByzhynar , fixed in 3190f78.

**Comment by Kozoriz** [comment]
Reviewed

**Comment by wolfylambova22** [comment]
Reviewed.

**Comment by AByzhynar** [comment]
Reviewed

**Comment by AByzhynar** [comment]
@LevchenkoS Can not merge it. Wrong branch name!

**Pull request closed without merging**